src/common/Throttle.cc

```diff
@@ -629,7 +654,9 @@ int OrderedThrottle::wait_for_ret() {
   complete_pending_ops();
 
   while (m_current > 0) {
+    ++waiters;
     m_cond.Wait(m_lock);
+    ++waiters;
```

Comment:
WELL, that's what I get for copying my diff without paying close enough attention.